src/main/groovy/de/gesellix/gradle/docker/models/DockerContainer.groovy

```diff
@@ -0,0 +1,393 @@
+package de.gesellix.gradle.docker.models
```

Comment:
Didn't think too much when I made the package, chose model because of reference to MVC, and the task itself is more the VC part of the equation. Assumed that time that we could have one for Image, etc., but thinking about it now, it is not likely needed ever, because Image build triggering, etc. are driven mostly by other gradle tasks already.

Don't mind renaming it, but it really represents the Container itself and you can create(), start(), stop(), etc. it, but also it has the declarative part where you expect it to be reloaded(), etc. maybe those methods could be renamed. It is not a service that would handle multiple containers, just the one.

One option is to abstract the service code to a real service, and just create a model which then internally calls the service, but creating too much abstraction here is a bit excessive in my opinion.

Maybe we should call it a DockerContainerFacade or DockerContainerLifecycleModel or something along those lines.